TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/console_link/services.yaml

```diff
@@ -2,21 +2,22 @@ source_cluster:
   endpoint: "https://capture-proxy:9200"
   allow_insecure: true
   basic_auth:
-      username: "admin"
-      password: "admin"
+    username: "admin"
+    password: "admin"
 target_cluster:
   endpoint: "https://opensearchtarget:9200"
   allow_insecure: true
   basic_auth:
-      username: "admin"
-      password: "myStrongPassword123!"
+    username: "admin"
+    password: "myStrongPassword123!"
 metrics_source:
   prometheus:
     endpoint: "http://prometheus:9090"
 backfill:
-    reindex_from_snapshot:
-        docker:
+  reindex_from_snapshot:
+    docker:
 snapshot:
   snapshot_name: "snapshot_2023_01_01"
-  s3_repo_uri: "s3://my-snapshot-bucket"
-  s3_region: "us-east-2"
+  s3:
```

Comment:
Should this have the fs keys to make snapshots with docker solution?

Reply:
Oh, yep, definitely. Fixing now.